packages/rxjs/src/index.ts

```diff
@@ -12,12 +12,13 @@
 /// <reference path="./testing/index.ts" />
 /* eslint-enable @typescript-eslint/triple-slash-reference */
 
+/* Configuration */
+export { config, GlobalConfig } from './internal/config.js';
```

Comment:
I believe this should be the other way around. We should probably remove `/internal/config.ts` file.

Reply:
I see, I will rollback the change and try to remove the `/internal/config.ts` file